daemon/cluster/cluster.go

```diff
@@ -280,27 +280,30 @@ func (c *Cluster) GetDataPathAddress() string {
 	return ""
 }
 
-// GetRemoteAddress returns a known advertise address of a remote manager if
+// GetRemoteAddressList returns the known advertise address of a remote managers if
 // available.
 // todo: change to array/connect with info
-func (c *Cluster) GetRemoteAddress() string {
+func (c *Cluster) GetRemoteAddressList() []string {
```

Comment:
ok was not sure what you meant with "info" so was leaving as reminder, but will do